src/app/cluster/cluster-details/cluster-details.component.ts

```diff
@@ -335,7 +335,7 @@ export class ClusterDetailsComponent implements OnInit, OnDestroy {
   }
 
   isSSHKeysEditEnabled(): boolean {
-    return MemberUtils.hasPermission(this._user, this._currentGroupConfig, 'sshKeys', Permission.Delete);
+    return MemberUtils.hasPermission(this._user, this._currentGroupConfig, 'sshkeys', Permission.Delete);
```

Comment:
Since you are already renaming it can you use enum here instead of hardcoding it again?